libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -0,0 +1,890 @@
+                                                    "fraudCheck", "synthetic_identity_check"),
+                                            Map.of(
+                                                    "txn", "RB000180729626",
+                                                    "checkMethod", "data",
+                                                    "fraudCheck", "impersonation_risk_check"),
+                                            Map.of(
+                                                    "checkMethod", "data",
+                                                    "activityFrom", "1963-01-01",
+                                                    "identityCheckPolicy", "none")))
+                            .build());
+
+    List<TestVc.TestEvidence> FRAUD_EVIDENCE_CRI_STUB_CHECK =
+            List.of(
+                    TestVc.TestEvidence.builder()
+                            .checkDetails(null)
+                            .type(CRI_STUB_CHECK_EVIDENCE_TYPE)
+                            .txn("some-uuid")
+                            .identityFraudScore(1)
+                            .build());
+
+    List<TestVc.TestEvidence> DCMAW_EVIDENCE =
+            List.of(
+                    TestVc.TestEvidence.builder()
+                            .txn("bcd2346")
+                            .strengthScore(3)
+                            .validityScore(2)
+                            .activityHistoryScore(1)
+                            .checkDetails(
+                                    List.of(
+                                            Map.of(
+                                                    "checkMethod", "vri",
+                                                    "identityCheckPolicy", "published",
+                                                    "activityFrom", "2019-01-01"),
+                                            Map.of(
+                                                    "checkMethod",
+                                                    "bvr",
+                                                    "biometricVerificationProcessLevel",
+                                                    3)))
+                            .build());
+
+    List<TestVc.TestEvidence> DCMAW_EVIDENCE_2 =
+            List.of(
+                    TestVc.TestEvidence.builder()
+                            .txn("cbf56c46-f33a-49de-8be0-c6318d8eecfc")
+                            .strengthScore(3)
+                            .validityScore(2)
+                            .activityHistoryScore(1)
+                            .checkDetails(
+                                    List.of(
+                                            Map.of(
+                                                    "checkMethod", "data",
+                                                    "identityCheckPolicy", "published",
+                                                    "activityFrom", "1982-05-23")))
```

Comment:
Interested how this differs from the DCMAW_EVIDENCE block above? It's not super clear from the names when you might use one over the other?